### PR TITLE
docs: adding search input in the left sidebar

### PR DESCRIPTION
**Pull request by alinais**
image

image

### CODE REVIEW

File: docs/src/components/Sidebar/Sidebar.tsx [hunk 1]
```diff
@@ -461,7 +486,12 @@ class Sidebar extends React.Component<any, any> {
       disabled: true,
     }
 
-    const withComponents = withPrototypes.concat(componentMenuItem).concat(menuItemsByType[0].items)
+    const shownMenuItems = query ? this.renderSearchItems() : menuItemsByType[0].items
+
+    const withComponents = withPrototypes
+      .concat(searchInput)
```
Comment (jurokapsiar) [inline docs/src/components/Sidebar/Sidebar.tsx]: Input cannot be in a menu - you would need to split it to multiple menus or move the input to somewhere else.

Comment (alinais) [reply docs/src/components/Sidebar/Sidebar.tsx]: we should include this explanation as part of the Do's and Dont's for the Menu

File: docs/src/components/Sidebar/Sidebar.tsx [hunk 2]
```diff
@@ -485,6 +499,18 @@ class Sidebar extends React.Component<any, any> {
           />
           <Text color="white" content={pkg.version} size="medium" styles={logoStyles} />
         </Segment>
+        <Menu vertical fluid pills styles={navBarStyles} items={withPrototypes} />
+        <Input
```
Comment (jurokapsiar) [inline docs/src/components/Sidebar/Sidebar.tsx:503]: did you consider using `<Dropdown>` component which already has the search functionality?

Comment (bmdalex) [reply docs/src/components/Sidebar/Sidebar.tsx:503]: +1 for `Dropdown`

**Comment by mnajdova** [comment]
Closing this in favor of https://github.com/stardust-ui/react/pull/1682

**Pull request closed without merging**